src/components/_global/BalModal/BalModal.vue

```diff
@@ -93,7 +93,8 @@ export default defineComponent({
 }
 
 .modal-card {
-  @apply w-full mx-auto h-full rounded-b-none sm:rounded-b-lg;
+  @apply max-w-full sm:max-w-lg md:max-w-md lg:max-w-sm;
+  @apply mx-auto h-full rounded-b-none sm:rounded-b-lg;
```

Comment:
This is a bit out of scope of the PR since it's a global change that affects all modals' width. Decided to bundle it here to match Pon's mockups for trade page (you can see that preview modal has the same width as trade card, which is not the case currently):
<img width="1004" alt="Screenshot 2021-05-19 at 15 46 24" src="https://user-images.githubusercontent.com/4247901/118814939-5f2a4b80-b8b9-11eb-8bce-397509bc2f42.png">

Imo it makes sense to make it global to keep modals as consistent as possible, and to align token selection modal with trade card as well.

Reply:
Yeah fair - I think its ok.